dist/opt/flight/etc/profile.d/90-locale-check.csh

```diff
@@ -0,0 +1,34 @@
+#==============================================================================
+# Copyright (C) 2020-present Alces Flight Ltd.
+#
+# This file is part of Flight Starter.
+#
+# This program and the accompanying materials are made available under
+# the terms of the Eclipse Public License 2.0 which is available at
+# <https://www.eclipse.org/legal/epl-2.0>, or alternative license
+# terms made available by Alces Flight Ltd - please direct inquiries
+# about licensing to licensing@alces-flight.com.
+#
+# Flight Starter is distributed in the hope that it will be useful, but
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, EITHER EXPRESS OR
+# IMPLIED INCLUDING, WITHOUT LIMITATION, ANY WARRANTIES OR CONDITIONS
+# OF TITLE, NON-INFRINGEMENT, MERCHANTABILITY OR FITNESS FOR A
+# PARTICULAR PURPOSE. See the Eclipse Public License 2.0 for more
+# details.
+#
+# You should have received a copy of the Eclipse Public License 2.0
+# along with Flight Starter. If not, see:
+#
+#  https://opensource.org/licenses/EPL-2.0
+#
+# For more information on Flight Starter, please visit:
+# https://github.com/openflighthpc/flight-starter
+#==============================================================================
+
+if ( ! $?flight_ROOT ) then
+  setenv flight_ROOT /opt/flight
+endif
+
+setenv flight_LOCALE_cmds "setenv LANG en_US.utf8\nsetenv LC_ALL en_US.utf8"
+bash --norc --noprofile $flight_ROOT/etc/profile.d/90-locale-check.sh
```

Comment:
Nice way to avoid scripting in `tcsh` :-)

Reply:
Life goals